model/patch_lifecycle.go

```diff
@@ -993,36 +984,20 @@ func AbortPatchesWithGithubPatchData(ctx context.Context, createdBefore time.Tim
 			continue
 		}
 
-		if p.IsCommitQueuePatch() {
```

Comment:
It looks like this if block used to apply to both the legacy commit queue and new merge queue patches. Is it sufficient for merge queue patches to just use `CancelPatch` here? Or is this logic never applicable to merge queue patches?